### DISCUSSION TITLE
AnimatedSprite component mandatory property 'texture'

### DISCUSSION THREAD

**Discussion by orliwiz** [mattjennings/svelte-pixi#17] [General]
Is there a reason that the 'texture' property is mandatory on the AnimatedSprite component? In your example [here](https://svelte-pixi.com/docs/components/animated-sprite) it isn't used, only 'textures'. My code seems to be working without it as well, i'm just getting a typescript error in vscode due to it.

I'm happy to submit a pull request to add the single '?' if that is what is wanted.

Thanks

**Comment by mattjennings** [comment]
oops, you're right, it shouldn't be. At one point it inherited the prop types of the Sprite component, so it must have been left over from that. I'll push a quick fix for it.

edit: fixed in [0.1.2](https://github.com/mattjennings/svelte-pixi/releases/tag/0.1.2)

**Reply by orliwiz** [reply]
awesome, thanks.
